R/controls.R

```diff
@@ -46,7 +42,9 @@ ionrangesliderLibs <- function() {
       script = "js/ion.rangeSlider.min.js",
       # ion.rangeSlider also needs normalize.css, which is already included in
       # Bootstrap.
-      stylesheet = c("css/ion.rangeSlider.css",
+      stylesheet = c(
+        "css/normalize.css",
```

Comment:
Was this added because `filter_slider()` looks bad without it when Bootstrap isn't present? It generally seems like a bad idea for the same reasoning that importing all of Bootstrap was a bad idea (it'll lead to conflicts with other CSS frameworks that want different "browser resets").

Even if `filter_slider()` looks bad without this, let's not worry about it for now (i.e., please revert this change). We should update ionrangeslider in a follow up PR anyway to match shiny's current version, which I think will solve this issue

Reply:
I included it because of the comment above it 

```r
# ion.rangeSlider also needs normalize.css, which is already included in
# Bootstrap.
```

I don't actually see a difference: reverted.